hudi-sync/hudi-dla-sync/src/main/java/org/apache/hudi/dla/DLASyncConfig.java

```diff
@@ -62,6 +62,9 @@
   @Parameter(names = {"--skip-ro-suffix"}, description = "Skip the `_ro` suffix for Read optimized table, when registering")
   public Boolean skipROSuffix = false;
 
+  @Parameter(names = {"--skip-rt-sync"}, description = "Skip the `_rt` suffix for Read optimized table, when registering")
```

Comment:
should be `Skip the `_rt` table syncing`

Reply:
done